packages/client/src/v2-events/features/events/actions/print-certificate/Review.tsx

```diff
@@ -179,19 +182,31 @@ export function Review() {
     ))
 
     if (confirmed) {
-      handleCertify?.()
-      actions.printCertificate.mutate({
-        eventId: fullEvent.id,
-        data: form,
-        transactionId: uuid(),
-        type: ActionType.PRINT_CERTIFICATE
-      })
-      clear()
-      navigate(ROUTES.V2.EVENTS.OVERVIEW.buildPath({ eventId }))
+      setPrintingInProgress(true)
+      actions.printCertificate.mutate(
+        {
+          eventId: fullEvent.id,
+          data: { ...formValues, templateId },
+          transactionId: uuid(),
+          type: ActionType.PRINT_CERTIFICATE
+        },
+        {
+          onSuccess: () => {
+            handleCertify?.()
+            navigate(ROUTES.V2.EVENTS.OVERVIEW.buildPath({ eventId }))
+          },
+          onError: (error) => {
+            throw new Error(error.message)
+          },
```

Comment:
```suggestion
```

What happens if this is omitted?

Reply:
Nothing happens. I think we can introduce tooltip/notification for notifying error.

Reply:
I expected the page being broke so that we can deal with it in a different ticket. But looks like Errorboundary only catches rendering errors. So added a todo comment here.